### DISCUSSION TITLE
Managing vendor email notifications

### DISCUSSION THREAD

**Discussion by deviantintegral** [ddev/ddev#5490] [DDEV Advisory Group]
We discussed on the advisory group today how there's annual tasks that need to be done to keep builds going. For example, we need to renew an Apple developer account every year to sign binaries. Docker requires us to re-apply each year for OSS hosting, and typically notifies us a week in advance by attempting to send a bill.

Currently, these notifications look to go to @rfay directly. We discussed setting up a group email account for accountability, but both @rfay and Cary Gordon mentioned previous organizational challenges they've had with this.

At Lullabot, we have a few guidelines we use for email addresses like this that help keep things running smoothly.

1. We only ever use groups (in our case, Google) and not forwarding aliases. That means that a copy of the email is kept in the group, and can be searched or viewed by team members even if they weren't part of the group when the email was sent.
2. When replying, we always keep the group email address in the CC list. That way the group has a record of all replies.
3. For notifications like domain expiries, two-step email tokens, and the like, we always respond to the group with a note. For example, "this was me resetting our password for <service> because they had a breach", or "we can safely let this domain expire as per <link to discussion>". That way, if no one responds to an important-looking email within a day or two, we know it's still to be done!

DDEV is using Zoho for email, and I'm guessing they have similar functionality. Who's best to investigate and set this up? Topics I see are:

1. What groups, and how many do we need? For example, I could see an accounts email for finances, something like operations for technical services, and perhaps `security@`. Or we might get away with a single `team@` group?
2. Does zoho support groups similar to google groups, that can allow external posting?
3. Who would be best to have on each email group?

**Comment by rfay** [comment]
Thanks for the good thinking here.

Zoho has groups, distribution lists, and shared mailboxes, which seem to do the things you're suggesting. 

New users cost $12/year, which is small in the scheme of things, and compared to Google Workspace. Of course Zoho has everything dialed in so you need just one more thing at every turn of your journey :)

<img width="280" alt="image" src="https://github.com/ddev/ddev/assets/112444/cbacebe7-a702-47c4-8017-9bdfe2519ef7">

**Comment by rfay** [comment]
I've moved this over to 
* https://github.com/ddev/ddev/issues/5491

because it's an item that needs attention, so closing the discussion here.
